this fails to build using nolibc because of either no _start_c or duplicate _start_c
--
--- a/libcontainer/dmz/Makefile
+++ b/libcontainer/dmz/Makefile
@@ -7,7 +7,7 @@
 # See nolibc supported arches in ./nolibc/arch-*.h
 NOLIBC_GOARCHES := 386 amd64 arm arm64 loong64 ppc64le riscv64 s390x
 
-ifneq (,$(filter $(GOARCH), $(NOLIBC_GOARCHES)))
+ifneq (0,0)
 	# We use the flags suggested in nolibc/nolibc.h, it makes the binary very small.
 	CFLAGS += -fno-asynchronous-unwind-tables -fno-ident -s -Os -nostdlib -lgcc
 else
